src/reseat/reseat.css

```
@@ -0,0 +1,19 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
```

you can combine the styles
```suggestion
h2 ,
p,
body {
  margin: 0;
}
```